### PR TITLE
UPDATE: Card accessibility

### PR DESCRIPTION
**Pull request by filipdanisko**
Updating card for accessibility, card is now possible navigate using keyboard and focus state is present.
<br/><br/><br/><url>LiveURL: https://orbit-components-update-card-a11y.surge.sh</url>

### CODE REVIEW

File: src/Card/CardSection/index.js [hunk 1]
```diff
@@ -90,12 +91,34 @@ class CardSection extends React.Component<any, Props> {
     }
   };
 
+  handleKeyPress = (ev: SyntheticKeyboardEvent<HTMLInputElement>) => {
+    if (ev.key === "Enter" || ev.key === " ") {
+      ev.preventDefault();
+      this.injectCallbackAndToggleSection();
+
+      const { onClose, onExpand, expanded } = this.props;
```
Comment (tomashapl) [inline src/Card/CardSection/index.js]: This code below does do the same as `injectCallbackAndToggleSection` please remove 🙏

File: src/Card/CardSection/index.js [hunk 2]
```diff
@@ -90,12 +91,34 @@ class CardSection extends React.Component<any, Props> {
     }
   };
 
+  handleKeyPress = (ev: SyntheticKeyboardEvent<HTMLInputElement>) => {
+    if (ev.key === "Enter" || ev.key === " ") {
```
Comment (tomashapl) [inline src/Card/CardSection/index.js]: In this case would be better to use `ev.code` it returns `Enter` or `Space`

Comment (filipdanisko) [reply src/Card/CardSection/index.js]: I tried `.code` before, not sure why but I'm getting `undefined` not sure what am I missing.

File: src/Card/CardSection/index.js [hunk 3]
```diff
@@ -91,20 +91,11 @@ class CardSection extends React.Component<any, Props> {
     }
   };
 
-  handleKeyPress = (ev: SyntheticKeyboardEvent<HTMLInputElement>) => {
-    if (ev.key === "Enter" || ev.key === " ") {
+  handleDownPress = (ev: SyntheticKeyboardEvent<HTMLDivElement>) => {
```
Comment (tomashapl) [inline src/Card/CardSection/index.js]: handleKeyDownPress 🙏

**Pull request merged**